src/base/templates/base.hbs

```diff
@@ -37,53 +32,11 @@
 
   <link rel="profile" href="//gmpg.org/xfn/11" />
 
-  <!--[if lt IE 9]>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/r29/html5.js"></script>
-    <script src="//css3-mediaqueries-js.googlecode.com/svn/trunk/css3-mediaqueries.js"></script>
-  <![endif]-->
-
 </head>
   <body class="{{#if config.map.geocoding_bar_enabled}} geocoding-bar-enabled{{/if}}">
+  <div id="site-wrap">
 
-  <header id="site-header" class="clearfix{{#if config.pages}} has-pages{{/if}}">
-    <h1 id="site-title">{{#block "site-title"}}{{/block}}</h1>
-
-    {{#block "pre_auth_nav"}}{{/block}}
-    <div id="auth-nav-container"></div>
-    {{#block "post_auth_nav"}}{{/block}}
-
-    {{#block "pre_pages_nav"}}{{/block}}
-    <div id="pages-nav-container"></div>
-    {{#block "post_pages_nav"}}{{/block}}
-
-    {{#if config.map.counter}}
```

Comment:
Note that the `counter` was not ported in this PR. It was only ever used on the raingardens flavor, I think, but we could port it at a later date if we need it again.